doc/source/dev/contributor/rendering_documentation.rst

```diff
@@ -69,6 +109,114 @@ on the cloud.
 #. Back in GitHub, at the bottom of the PR, select “Show all Checks”.
 #. Next to “ci/circleci: build_docs artifact”, select “Details”.
 
+.. _docs-guidelines:
+
+Documentation Guidelines
+------------------------
+
+Use "must", not "should"
+~~~~~~~~~~~~~~~~~~~~~~~~
+When specifying a required condition on the input parameters, the
+word "must" is preferable to "should".  For many English speakers,
+"must" implies a stronger constraint than "should",  e.g. "I must
+have oxygen to live" versus "I should exercise more".
+
+    Yes::
+
+            Parameters
+            ----------
+            x : float
+                `x` must be nonnegative.
+
+    No::
+
+            Parameters
+            ----------
+            x : float
+                `x` should be nonnegative.
```

Comment:
Interesting, `git grep -E -A 4 "Parameters" | grep -E -i "should" | wc -l` returns about 50 results.